domains/Contact/ManageContactFeed/Web/ViewHelpers/Actions/ActionFeedContactInformation.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Contact\ManageContactFeed\Web\ViewHelpers\Actions;
+
+use App\Models\ContactFeedItem;
+
+class ActionFeedContactInformation
+{
+    public static function data(ContactFeedItem $item): array
+    {
+        $contact = $item->contact;
+        $contactInformation = $item->feedable;
+
+        return [
+            'information' => [
+                'object' => $contactInformation ? [
+                    'id' => $contactInformation->id,
+                    'label' => $contactInformation->name,
+                    'data' => $contactInformation->contactInformationType->protocol ? $contactInformation->contactInformationType->protocol.$contactInformation->data : $contactInformation->data,
```

Comment:
you should be able to write it, but I did not tested it:
```suggestion
                    'data' => optional($contactInformation->contactInformationType->protocol).$contactInformation->data,
```

Reply:
Unfortunately that doesn't work. I understand the intention though.